share/ipld/nmt_adder.go

```diff
@@ -60,3 +56,26 @@ func (n *NmtNodeAdder) Commit() error {
 
 	return n.add.Commit()
 }
+
+// MaxSizeBatchOption sets the maximum amount of buffered data before writing
+// blocks.
+func MaxSizeBatchOption(size int) ipld.BatchOption {
+	return ipld.MaxSizeBatchOption(BatchSize(size))
+}
+
+// BatchSize calculates the amount of nodes that are generated from block of 'squareSizes'
+// to be batched in one write.
+func BatchSize(squareSize int) int {
```

Comment:
This has to be updated because we remov one additional layer. Hmm, I think there was a test verifying its correctness. Is it still there?

Reply:
As per the sync discussion with Vlad, the calculations are correct here. The comments still needs to be updated though